luigi_pipeline/lib/model/mito_mt_schema.py

```diff
@@ -0,0 +1,268 @@
+        return vep.get_expr_for_vep_protein_domains_set_from_sorted(
+            self.mt.sortedTranscriptConsequences)
+
+    @row_annotation(name='transcriptConsequenceTerms', fn_require=sorted_transcript_consequences)
+    def transcript_consequence_terms(self):
+        return vep.get_expr_for_vep_consequence_terms_set(self.mt.sortedTranscriptConsequences)
+
+    @row_annotation(name='transcriptIds', fn_require=sorted_transcript_consequences)
+    def transcript_ids(self):
+        return vep.get_expr_for_vep_transcript_ids_set(self.mt.sortedTranscriptConsequences)
+
+    @row_annotation(name='mainTranscript', fn_require=sorted_transcript_consequences)
+    def main_transcript(self):
+        return vep.get_expr_for_worst_transcript_consequence_annotations_struct(
+            self.mt.sortedTranscriptConsequences)
+
+    @row_annotation(name='geneIds', fn_require=sorted_transcript_consequences)
+    def gene_ids(self):
+        return vep.get_expr_for_vep_gene_ids_set(self.mt.sortedTranscriptConsequences)
+
+    @row_annotation(name='codingGeneIds', fn_require=sorted_transcript_consequences)
+    def coding_gene_ids(self):
+        return vep.get_expr_for_vep_gene_ids_set(self.mt.sortedTranscriptConsequences, only_coding_genes=True)
+
+    @row_annotation()
+    def gnomad(self):
+        return self._selected_ref_data.gnomad_mito
+
+    @row_annotation()
+    def mitomap(self):
+        return self._selected_ref_data.mitomap
+
+    @row_annotation(name='APOGEE')
+    def mitimpact(self):
+        return self._selected_ref_data.mitimpact
+
+    @row_annotation(name='HmtVar')
+    def hmtvar(self):
+        return self._selected_ref_data.hmtvar
+
+    @row_annotation()
+    def helix(self):
+        return self._selected_ref_data.helix_mito
+
+    @row_annotation()
+    def clinvar(self):
+        return self._selected_ref_data.clinvar_mito
+
+    @row_annotation()
+    def dbnsfp(self):
+        return self._selected_ref_data.dbnsfp_mito
+
+
+class SeqrMitoVariantSchema(SeqrMitoSchema):
+
+    @row_annotation(name='AC')
+    def ac(self):
+        return self.mt.AC_hom
+
+    @row_annotation(name='AF')
+    def af(self):
+        return self.mt.AF_hom
+
+    @row_annotation(name='AN')
+    def an(self):
+        return self.mt.AN
+
+    @row_annotation(name='AC_het')
+    def ac_het(self):
+        return self.mt.AC_het
+
+    @row_annotation(name='AF_het')
+    def af_het(self):
+        return self.mt.AF_het
+
+    @row_annotation()
+    def common_low_heteroplasmy(self):
+        return self.mt.common_low_heteroplasmy
+
+    @row_annotation()
+    def hap_defining_variant(self):
+        return self.mt.hap_defining_variant
+
+    @row_annotation()
+    def mitotip_trna_prediction(self):
+        return self.mt.mitotip_trna_prediction
+
+    @row_annotation()
+    def high_constraint_region(self):
+        return hl.is_defined(self._high_constraint_region[self.mt.locus])
+
+
+class SeqrMitoGenotypesSchema(BaseMTSchema):
```

Comment:
This can probably inherit from `SeqrGenotypesSchema` instead and just override the methods like `_genotype_fields` that are actually different